tests/test_job.py

```diff
@@ -443,8 +443,47 @@ def test_job_11_start_training_round(self,
         self.assertListEqual(nodes, ['node-1'])
         self.assertEqual(serialize_load_patch.call_count, 1)
 
-    def test_job_12_update_parameters_with_invalid_arguments(self):
-        """ Testing update_parameters method with invalid arguments.s"""
+    def test_job_12_start_nodes_training_round_optim_aux_var(self):
+        """Test that 'optim_aux_var' is properly used in 'start_nodes_training_round'."""
+        mock_aux_var = {"module": {"key": "val"}}
+        # General setup: skip requests sending and replies processing.
```

Comment:
I have a very minor concern about the name `mock_aux_var`: it can be a bit confusing since it is not really a `Mock` object

Reply:
OK, I renamed it to `fake_aux_var`.